Add missing include to avoid build error: "shared_ptr in namespace std does not name a template type"
Author: Samuel Moors (Vrije Universiteit Brussel)
diff -ur egttools-0.1.10.dev2.orig/include/egttools/finite_populations/behaviors/NFGStrategies.hpp egttools-0.1.10.dev2/include/egttools/finite_populations/behaviors/NFGStrategies.hpp
--- egttools-0.1.10.dev2.orig/include/egttools/finite_populations/behaviors/NFGStrategies.hpp	2022-08-06 16:35:37.000000000 +0200
+++ egttools-0.1.10.dev2/include/egttools/finite_populations/behaviors/NFGStrategies.hpp	2022-08-12 17:22:35.065010288 +0200
@@ -25,6 +25,7 @@
 
 #include <algorithm>
 #include <egttools/finite_populations/behaviors/AbstractNFGStrategy.hpp>
+#include <memory>
 #include <random>
 #include <stdexcept>
 
